source/system-administrators/index.rst

```diff
@@ -63,8 +63,8 @@ Upgrading Crafter CMS
    :maxdepth: 1
    :titlesonly:
 
-   upgrade/upgrading-to-craftercms-3-1-0.rst
    upgrade/index.rst
+   upgrade/upgrading-to-craftercms-3-1-0.rst
```

Comment:
We have removed the link to the instructions for upgrading to Crafter CMS 3.1.0 from the table of contents as the instructions is specifically only if you are upgrading to version 3.1.0, which is now old, as we have a newer version, 3.1.1 with an updated instruction on how to upgrade.